### PR TITLE
Update cockpit new UI

### PR DESCRIPTION
**Pull request by peterdragun**
Adding option to use old UI which is used in RHEL7 and new UI that is used in both RHEL8 and 9.

### CODE REVIEW

**Comment by peterdragun** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k "test_positive_cockpit[rhel7] or test_positive_cockpit[rhel8]"
airgun: 676

File: airgun/entities/host.py [hunk 1]
```diff
@@ -243,26 +243,34 @@ def set_puppet_class_parameter_value(self, entity_name, name, value):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
-    def get_webconsole_content(self, entity_name):
+    def get_webconsole_content(self, entity_name, use_new_ui=True):
         """Navigate to host's webconsole and return the hostname from the cockpit page
 
         :param str entity_name: The host name for which to set the parameter value.
+        :param bool use_new_ui: When true: use new UI elements for RHEL8 and RHEL9.
```
Comment (tstrych) [inline airgun/entities/host.py:250]: I would maybe use int instead of bool, non-blocking

**Review by tstrych — APPROVED** [review]
ACK

File: airgun/entities/host.py (hunk 1 above)
Comment (peterdragun) [reply airgun/entities/host.py]: Updated to use RHEL version instead of bool. This would be also easier to use from robottelo

**Review by pondrejk — APPROVED** [review]

**Pull request merged**